docs/source/v2/central-app.blade.md

```diff
@@ -17,6 +17,8 @@ Routes in the `routes/web.php` file are the central routes. When they are visite
 
 However, since you don't want routes related to the app on your main domain and sign up forms on tenant domains, you must also define what domains host the central stuff in the `tenancy.exempt_domains` config.
 
+The exempt domains should not include the protocol. For example, you should include ```example.com``` rather than ```https://example.com```, otherwise this will throw a ```TenantCouldNotBeIdentifiedException``` exception.
```

Comment:
```suggestion
The exempt domains should not include the protocol. For example, you should include ```example.com``` rather than ```https://example.com```.
```

Reply:
I don't think that's necessary.

Reply:
Sounds good.